src/openrct2/core/FileWatcher.h

```diff
@@ -59,7 +59,9 @@ class FileWatcher
     ~FileWatcher();
 
 private:
+#if defined(_WIN32) || defined(__linux__)
```

Comment:
Is the check for `defined(_WIN32)` really necessary here? `_finished` only seems to be used inside `__linux__` preprocessor blocks in the rouce file.